docs/cheatsheet.rst

```diff
@@ -6,12 +6,13 @@ Cheatsheet
 
 Order of Precedence of Operators
 ================================
+
 .. include:: types/operator-precedence-table.rst
 
 .. index:: assert, block, coinbase, difficulty, prevrandao, number, block;number, timestamp, block;timestamp, msg, data, gas, sender, value, gas price, origin, revert, require, keccak256, ripemd160, sha256, ecrecover, addmod, mulmod, cryptography, this, super, selfdestruct, balance, codehash, send
```

Comment:
These index entries should be split to so that each is placed next to sections that talks about its topic.

Reply:
Not sure about this. It would be great it you can brief me more about it.

Reply:
The [`index` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#index-generating-markup) adds entries to [keyword index](https://docs.soliditylang.org/en/latest/genindex.html). As it is now, all of them are pointing at a section called `ABI Encoding and Decoding Functions`, which seems wrong. You need to have separate `index` directive for each section, with entries matching what's actually inside that section.

Reply:
Added the indices.